theano/tensor/tests/test_basic.py

```diff
@@ -1577,10 +1577,6 @@ def test_py_c_match():
     integers=(randint(2, 3), randint(2, 3)),
     int8=[numpy.arange(-127, 128, dtype='int8'),
           numpy.arange(-127, 128, dtype='int8')[:, numpy.newaxis]],
-    uint8=[numpy.arange(0, 255, dtype='uint8'),
-           numpy.arange(0, 255, dtype='uint8')[:, numpy.newaxis]],
-    uint16=[numpy.arange(0, 65535, dtype='uint16'),
-            numpy.arange(0, 65535, dtype='uint16')[:, numpy.newaxis]],
```

Comment:
I tested it rapidly in numpy and it work. I think the problem is that numpy will return float16 for this case and not float32 for uint8. But why did you remove uint16 that return float32?

>>> np.arctan2(np.asarray([1], dtype='uint8'), np.asarray([1], dtype='uint8'), )
array([ 0.78515625], dtype=float16)